src/quo2/components/notifications/toast.cljs

```diff
@@ -0,0 +1,88 @@
+(ns quo2.components.notifications.toast
+  (:require
+   [i18n.i18n :as i18n]
+   [quo2.components.icon :as icon]
+   [quo2.components.markdown.text :as text]
+   [quo2.components.notifications.count-down-circle :as count-down-circle]
+   [quo2.foundations.colors :as colors]
+   [quo2.theme :as theme]
+   [react-native.core :as rn]))
+
+(def ^:private themes
+  {:container        {:light {:background-color colors/neutral-80-opa-80}
+                      :dark  {:background-color colors/white-opa-70}}
+   :text             {:light {:color colors/white}
+                      :dark  {:color colors/neutral-100}}
+   :icon             {:light {:color colors/white}
+                      :dark  {:color colors/neutral-100}}
+   :action-container {:light {:background-color :colors/neutral-80-opa-5}
+                      :dark  {:background-color :colors/white-opa-5}}})
+
+(defn- merge-theme-style
+  [component-key styles]
+  (merge (get-in themes [component-key (theme/get-theme)]) styles))
+
+(defn toast-action-container
```

Comment:
why we have this as separate function?

Reply:
The idea is that future actions can use this to wrap the real action component